packages/builder/src/components/common/CodeEditor/CodeEditor.svelte

```diff
@@ -423,6 +423,7 @@
     })
     isAIGeneratedContent = true
     dispatch("change", code)
+    dispatch("ai_suggestion")
```

Comment:
It's much cleaner to just dispatch a new event than try and add modify the change object